File: android/src/ru/deadsoftware/cavedroid/AndroidLauncher.java
package ru.deadsoftware.cavedroid;

import android.content.pm.PackageManager;
import android.os.Bundle;
import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;

public class AndroidLauncher extends AndroidApplication {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
        config.useImmersiveMode = true;
        String gameFolder = "";
        try {
            gameFolder = getPackageManager().getPackageInfo(getPackageName(), 0).applicationInfo.dataDir;
        } catch (PackageManager.NameNotFoundException e) {
            e.printStackTrace();
            exit();
        }
        CaveGame caveGame = new CaveGame(gameFolder, true, null);
        caveGame.setDebug(BuildConfig.DEBUG);
        initialize(caveGame, config);
    }

    @Override
    public void onBackPressed() {
    }
}

File: android/src/main/kotlin/ru/deadsoftware/cavedroid/AndroidLauncher.kt
package ru.deadsoftware.cavedroid

import android.os.Bundle
import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration

class AndroidLauncher : AndroidApplication() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val gameDataDirectoryPath = packageManager.getPackageInfo(packageName, 0)
            .applicationInfo.dataDir;

        val config = AndroidApplicationConfiguration()
        config.useImmersiveMode = true

        initialize(
            /* listener = */ CaveGame(
                gameDataDirectoryPath = gameDataDirectoryPath,
                isTouchScreen = true,
                isDebug = BuildConfig.DEBUG,
                preferencesStore = AndroidPreferencesStore(applicationContext)
            ),
            /* config = */ config
        )
    }

    @Suppress("OVERRIDE_DEPRECATION")
    override fun onBackPressed() {
        // ignore
    }

}